handler/job/status.go

```diff
@@ -37,7 +38,7 @@ type SourceStatus struct {
 func (DefaultHandler) GetStatusHandler(ctx context.Context, db *gorm.DB, id string) ([]SourceStatus, error) {
 	db = db.WithContext(ctx)
 	var preparation model.Preparation
-	err := preparation.FindByIDOrName(db, id)
+	err := preparation.FindByIDOrName(db, id, "OutputStorages")
```

Comment:
maybe make this a `const`

Reply:
We're using those literals everywhere unfortunately.